utils/statistics-flow-reduce-operators.js

```diff
@@ -0,0 +1,108 @@
+'use strict';
+
+/**
+ * It's about reduction
+ *
+ * FROM:
+ *
+ * [date][serviceName].businessProcess[status] = num;
+ * [date][serviceName].certificate[name][status] = num;
+ * [date][serviceName].processingStep[stepName].pending.businessProcess = num;
+ * [date][serviceName].processingStep[stepName].pending.certificate[name] = num;
+ * [date][serviceName].processingStep[stepName].byProcessor[processorId][status].businessProcess =
+ * num;
+ * [date][serviceName].processingStep[stepName].byProcessor[processorId][status].certificate[name] =
+ * num;
+ *
+ * TO:
+ * [date][processorId] = {
+ *  processor: processorId,
+ *  date: "YYYY-MM-DD",
```

Comment:
Why do we provide `processor` and `date` on this object? This information is on keys